Mathlib/RingTheory/HahnSeries/Basic.lean

```diff
@@ -537,4 +537,31 @@ theorem order_ofForallLtEqZero [Zero Γ] (f : Γ → R) (hf : f ≠ 0) (n : Γ)
 
 end LocallyFiniteLinearOrder
 
+section Truncate
+variable [Zero R]
+
+/-- Zeroes out coefficients of a `HahnSeries` at indices not less than `c`. -/
```

Comment:
It seems the singular form of the verb "zero" is "zeros": https://en.wiktionary.org/wiki/zeros_in